### PR TITLE
feat: played with mark tag

### LINKED ISSUE #95: the `<mark>` tag

### PR DESCRIPTION
**Pull request by Alez05**
fix #95

### CODE REVIEW

**Review by flivanov92 — APPROVED** [review]
👍

**Review by antonia2203 — APPROVED** [review]
💯

File: src/alez05/mark.pug [hunk 1]
```diff
@@ -0,0 +1,25 @@
+html
+  head
+    title ∆ƒ∆®ƒ^∆ƒ∆®ƒ
+  body
+    div
+      h1 The mark element
+    div`
```
Comment (MariaTeo) [inline src/alez05/mark.pug]: you don't need the back-tick

**Review by MariaTeo — CHANGES_REQUESTED** [review]

**Review by Vivy — APPROVED** [review]
Good Naruto ref

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**